src/middleware/packages/ldp/services/resource/actions/get.js

```diff
@@ -37,6 +37,10 @@ module.exports = {
       aclVerified: { type: 'boolean', optional: true }
     },
     cache: {
+      enabled: function(ctx) {
+        //On regarde si le containerURI est celui d'un fichier, pas de cache si c'est le cas
+        return /[^/]*$/.exec(ctx.options.parentCtx.params.containerUri)[0] == 'files' ? false : true;
```

Comment:
On essaie de toujours utiliser `===` au lieu de `==`.
En l'occurence cette ligne peut être simplifiée comme ça:
```suggestion
        return /[^/]*$/.exec(ctx.options.parentCtx.params.containerUri)[0] !== 'files';
```